app/models/conversation.rb

```diff
@@ -14,4 +14,11 @@ def includes?(user)
     sender == user || recipient == user
   end
 
+  def self.chattiest
+    select("conversations.*, COALESCE(COUNT(messages.id), 0) AS message_count")
+    .joins("LEFT JOIN messages ON messages.conversation_id = conversations.id")
+    .group("conversations.id")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.